src/analytics/analytics-handler-interface.ts

```diff
@@ -0,0 +1,69 @@
+import type { AnalyticsEvent } from '@internetarchive/analytics-manager';
+
+export interface AnalyticsHandlerInterface {
+  /**
+   * A general purpose analytics ping that takes arbitrary key-value pairs
+   * and pings the analytics endpoint
+   *
+   * @param {Record<string, any>} values
+   */
+  sendPing(values: Record<string, any>): void;
+
+  /**
+   * Send a sampled event
+   *
+   * @param {options} AnalyticsEvent
+   */
+  sendEvent(options: AnalyticsEvent): void;
+
+  /** @deprecated use sendEvent instead */
+  send_event(
```

Comment:
Q: Curious about the need for this deprecated method. Does collection-browser have subcomponents that still expect this?

Reply:
i just copy pasta this from class from offshoot... we can probably remove it here then

Reply:
removing this file